cmd/manager/main.go

```diff
@@ -26,17 +29,28 @@ import (
 	"github.com/open-policy-agent/gatekeeper/pkg/upgrade"
 	"github.com/open-policy-agent/gatekeeper/pkg/webhook"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
-	"os"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/go-logr/zapr"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 func main() {
+	var (
+		debug = flag.Bool("debug", false, "Enable debug and info level logging")
```

Comment:
`--debug` may be confusing as it could also refer to the behavior of the program, maybe `--log-level-debug` or something similar?

We should choose a name that allows us to add more granular logging thresholds in the future.